pkg/model/components/clusterautoscaler.go

```diff
@@ -43,6 +43,8 @@ func (b *ClusterAutoscalerOptionsBuilder) BuildOptions(o interface{}) error {
 		v, err := util.ParseKubernetesVersion(clusterSpec.KubernetesVersion)
 		if err == nil {
 			switch v.Minor {
+			case 24:
+				image = "registry.k8s.io/autoscaling/cluster-autoscaler:v1.23.0"
```

Comment:
This should probably go separately.

Reply:
Agree. https://github.com/kubernetes/kops/pull/13510